src/myWallet/wallet.lightning.service.js

```
@@ -0,0 +1,407 @@
+            SendPaymentMethod_Tags.SparkInvoice
+        ) {
+            console.log(
+                `Token ID: ${prepareSendResponse.paymentMethod.inner.tokenIdentifier}`,
+            );
+            console.log(
+                `Fees: ${prepareSendResponse.paymentMethod.inner.fee} token base units`,
+            );
+        }
+
+        // Send the token payment
+        const sendResponse = await sdk.sendPayment({
```

ESLint

[error] Delete `····`

(prettier/prettier)